src/HealthChecks.MongoDb/HealthChecks.MongoDb.csproj

```
@@ -1,7 +1,8 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>$(DefaultLibraryTargetFrameworks)</TargetFrameworks>
+    <!-- Match the TFMs of MongoDB.Driver -->
+    <TargetFrameworks>$(DefaultNetCoreApp);netstandard2.1;net472</TargetFrameworks>

```

Perhaps we should simply drop the NS support?

```suggestion
    <TargetFrameworks>$(DefaultNetCoreApp);net472</TargetFrameworks>

```

---

Since the underlying `MongoDB.Driver` library supports `netstandard2.1`, I feel like the health check library should as well. It would be a pity to block someone for no reason.

For example, I think Unity supports netstandard2.1 - https://docs.unity3d.com/6000.0/Documentation/Manual/dotnet-profile-support.html